ceph.spec.in

```diff
@@ -159,7 +164,15 @@ BuildRequires:	gperftools-devel >= 2.6.1
 BuildRequires:	gperftools-devel >= 2.4
 %endif
 %endif
+%if 0%{?qa_run}
 BuildRequires:  jq
+%if 0%{?fedora} || 0%{?rhel}
```

Comment:
@tchaikov What about Ubuntu?

Reply:
please see the changes in `debian/control` in the very same commit.